projects/plugins/boost/tests/e2e/lib/env/prerequisites.js

```diff
@@ -142,19 +126,16 @@ export async function ensureConnectedState( requiredConnected = undefined, page
 }
 
 export async function connect( page ) {
-	logger.prerequisites( `Connecting Boost plugin to WP.com` );
-	// Boost cannot be connected to WP.com using the WP-CLI because the site is considered
-	// as a localhost site. The only solution is to do it via the site itself running under the localtunnel.
 	const jetpackBoostPage = await JetpackBoostPage.visit( page );
-	await jetpackBoostPage.connect();
+	await jetpackBoostPage.chooseFreePlan();
 	await jetpackBoostPage.isOverallScoreHeaderShown();
 }
 
 export async function disconnect() {
 	logger.prerequisites( `Disconnecting Boost plugin to WP.com` );
-	const cliCmd = 'jetpack-boost connection deactivate';
```

Comment:
Notice this in your review @thingalon, The reason I am doing this is because, if there are other jetpack plugins installed, boosts cli commands aborts the disconnect, which `jetpack disconnect blog` forces the disconnect.